rollup.config.js

```diff
@@ -0,0 +1,77 @@
+
+import nodeResolve from 'rollup-plugin-node-resolve';
+import commonjs    from 'rollup-plugin-commonjs';
+
```

Comment:
I don't think this file got linted.

Reply:
correct.  there are 1400 linting errors because i forgot to put the linter in the build.

fixing promptly

![image](https://user-images.githubusercontent.com/77482/116465537-349d2380-a822-11eb-9a99-57799c25117f.png)

Reply:
With permission I am requesting to land the linter after this because of complexities around that eslint handles `commonjs` and `es6` differently

Probably same-day